tools/StaticAnalysis/Exceptions/Az.OperationalInsights/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,5 @@
+AssemblyFileName,ClassName,Target,Severity,ProblemId,Description,Remediation
+Az.OperationalInsights,Microsoft.Azure.Commands.OperationalInsights.Tables.MigrateAzureOperationalInsightsTableCommand,Migrate-AzOperationalInsightsTable,1,8300,"Migrate-AzOperationalInsightsTable uses the verb 'Migrate', which is not on the list of approved verbs for PowerShell commands. Use the cmdlet 'Get-Verb' to see the full list of approved verbs and consider renaming the cmdlet.",Consider renaming the cmdlet to use an approved verb for PowerShell.
+Az.OperationalInsights,Microsoft.Azure.Commands.OperationalInsights.Clusters.GetAzureOperationalInsightsClusterCommand,Get-AzOperationalInsightsCluster,0,3000,The type of property 'KeyRsaSize' of type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSKeyVaultProperties' has changed from 'System.Int32' to 'System.Nullable`1[System.Int32]'.,Change the type of property 'KeyRsaSize' back to 'System.Int32'.
+Az.OperationalInsights,Microsoft.Azure.Commands.OperationalInsights.Clusters.GetAzureOperationalInsightsClusterCommand,Get-AzOperationalInsightsCluster,0,3010,The property 'MaxCapacity' of type 'Microsoft.Azure.Management.OperationalInsights.Models.CapacityReservationProperties' has been removed.,Add the property 'MaxCapacity' back to type 'Microsoft.Azure.Management.OperationalInsights.Models.CapacityReservationProperties'.
```

Comment:
this is a breaking change, you need to create a new class PSCapacityReservationProperties to wrap https://github.com/Azure/azure-powershell/blob/358bfeaf7f91a28a68d5b7adf3ac2dec151c46de/src/OperationalInsights/OperationalInsights/Models/PSCluster.cs#L108

Reply:
added PSCapacityReservationProperties.cs